packages/client/components/AzureDevOpsScopingSearchResultItem.tsx

```diff
@@ -1,111 +0,0 @@
-import styled from '@emotion/styled'
```

Comment:
This wasn't being used anywhere as we're using `ScopingSearchResultItem`, so I've removed it